wpiutil/src/main/java/edu/wpi/first/util/datalog/DataLogEntry.java

```diff
@@ -6,15 +6,39 @@
 
 /** Log entry base class. */
 public class DataLogEntry {
+  /**
+   * Constructs a data log entry.
+   *
+   * @param log datalog
+   * @param name name of the entry
+   * @param type Data type
+   * @param metadata metadata
+   * @param timestamp entry creation timestamp (0=now)
+   */
   protected DataLogEntry(DataLog log, String name, String type, String metadata, long timestamp) {
     m_log = log;
     m_entry = log.start(name, type, metadata, timestamp);
   }
 
+  /**
+   * Constructs a data log entry.
+   *
+   * @param log datalog
+   * @param name name of the entry
+   * @param type Data type
+   * @param metadata metadata
+   */
   protected DataLogEntry(DataLog log, String name, String type, String metadata) {
     this(log, name, type, metadata, 0);
   }
 
+  /**
+   * Constructs a data log entry.
+   *
+   * @param log datalog
+   * @param name name of the entry
+   * @param type Data type
+   */
```

Comment:
Maybe add a note about the default values for parameters that aren't included (timestamp is now, metadata is empty)? Same for all the other constructors except for the base constructor.

Reply:
Looking over the previous existing documentation for similar methods they do not appear to include this additional detail within the comments. I kept a similar style to these. Is this change something worth breaking into another PR at some point vs in place here?. There would be changes outside of the classes I just added the missing Javadoc to.

Reply:
Yeah, it can go in another PR. Teams can always use VS Code to control-click the constructor and see default values. That said, in non-IDE contexts or tooltips, it would be convenient to see the default parameter values.